content/300-guides/200-deployment/110-deployment-guides/400-deploying-to-aws-lambda.mdx

```diff
@@ -334,8 +334,11 @@ The Serverless configuration file includes a package pattern that excludes all P
 ```
 package:
   patterns:
+    - '!node_modules/@prisma/engines/**'
     - '!node_modules/.prisma/client/libquery_engine-*'
     - 'node_modules/.prisma/client/libquery_engine-rhel-*'
+    - '!node_modules/prisma/libquery_engine-*'
+    - 'node_modules/prisma/libquery_engine-rhel-*'
```

Comment:
```suggestion
    - '!node_modules/.prisma/client/libquery_engine-*'
    - 'node_modules/.prisma/client/libquery_engine-rhel-*'
    - '!node_modules/prisma/libquery_engine-*'
    - 'node_modules/prisma/libquery_engine-rhel-*'
    - '!node_modules/@prisma/engines/**'
```
Let's change the order a bit:
- First two lines about Prisma Client that should indeed contains no query engines besides the Red Hat one.
- Last line about all the Engines in @prisma/engines that are not required.

Why do you want to include the Query Engine of the CLI though?

Reply:
@janpio You're saying that `node_modules/prisma/libquery_engine-rhel-*` is not needed? I didn't know it is for the CLI, I was just blindly including everything Red Hat. If so, I will incorporate your reordering suggestion and simplify further.

Reply:
I don't think it is, unless the `prisma` CLI is invoked and included. `node_modules/prisma` itself is only present when the `prisma` package is installed, and probably the Engine file is copied over there at some point in the process.